docs/repo-splitting.md

```diff
@@ -0,0 +1,41 @@
+# Splitting into Repositories
+
+## Background
+
+The [Reviewer](https://github.com/libero/reviewer) project is currently a mono repo.
+It has been decided that splitting the repository is necessary for scaling the project. There have been a number of issues raised to do this splitting and this document is being written to ensure that this is done in a consistent way.
+
+## Method
+
+### Create a new repository
+
+- Make sure you have the permissions and create a repository with the correct name, if in doubt - talk.
+- Ensure the new repository has the "eLife developers" team added to it with `Maintain` access.
+
+### Move the code
+
+:exclamation: It's important that the history of the files is kept during the move.
+
+- Read through and then use the [Splitting a subfolder out into a new repository](https://help.github.com/en/github/using-git/splitting-a-subfolder-out-into-a-new-repository) document.
+- Any questions ask someone who's done it before, like @will-byrne
+
+### Test & Infrastructure
+
+Ensure the following:
+
+- the package is linted
```

Comment:
Is this an appropriate time to move all code to `eslint` and have a consistent (where possible) configuration? Or can this be handled as a followup to each appropriate project?